index.html

```diff
@@ -1261,66 +1261,68 @@ <h2>Characters</h2>
-		   <span lang=hi>&#x092F;&#x0942;&#x0928;&#x093F;&#x0915;&#x094B;&#x0921;</span>
-		</p>
-		<p>The last example, in the Devanagari script (which is used to write Hindi, among other languages) is made up of four graphemes:</p>
-		<p class=bigtext lang=hi>&#x092F;&#x0942;&nbsp;&#x0928;&#x093F;&nbsp;&#x0915;&#x094B;&nbsp;&#x0921;</p>
+    <p>A <dfn class="lint-ignore">visual text unit</dfn> is a single visible "character" of text. This can be on a screen or in other media, such as printed on paper or written on the back of a napkin. Such units are a matter of perception and the term is necessarily imprecise, particularly when it comes to writing systems that use a combination of combining marks, complex positioning, or complex shaping based on context. A [=visual text unit=] generally shouldn't be broken apart by operations such as line-breaking or hyphenation.</p>
+    
+    <p>There are some other terms with similar meanings. The term [=grapheme=] is sometimes used by linguists (and others) to refer to what a typical user would perceive as a single unit. The term [=glyph=] is used by typographers (and others) to refer to an individual rendering unit. Sometimes a [=visual text unit=] is said to be a <dfn class="lint-ignore">user-perceived character</dfn>, although this name can be misleading, as some users in certain scripts will "perceive" (read) a particular shape in the text as separate "characters", such as when certain South Asian scripts use "stacks".</p>
+    
+    <p>At their simplest, [=visual text units=] are a single shape ([=glyph=]) that can be tied one-to-one to a "character" in the computer's memory. But a visual text unit can also be formed from a sequence of characters.</p>
+    
+    <aside class="example" title="Examples of visual text units" id="visual-text-unit-example">
+		<p>Here is the word for "Unicode" in the English language (which uses the Latin script). It is made up of seven Unicode characters that form seven visual text units:</p>
+		<p class="bigtext" style="margin-block-start: 12px; margin-block-end: 12px">Unicode</p>
+		<p>Here is the word for "Unicode" in the Hindi language (which uses the Devanagari script):</p>
+        <p><span class="bigtext" lang="hi">&#x092F;&#x0942;&#x0928;&#x093F;&#x0915;&#x094B;&#x0921;</span></p>
+        <p>This word also happens to be made up of seven Unicode characters, but these characters form only four visual text units (indicated here with color):</p>
+		<p><span class="bigtext" lang="hi">&#x092F;&#x0942;&nbsp;<span style="color:red">&#x0928;&#x093F;</span>&nbsp;<span style="color:black">&#x0915;&#x094B;</span>&nbsp;<span style="color:magenta">&#x0921;</span></p>
     </aside>
+	
+    <p>[=Visual text units=] are the visual representation of an underlying character sequence encoded in the computer's memory. In order to encode characters for use by software, we have to start by agreeing on what is being encoded.</p>
```

Comment:
I think that the first sentence essentially repeats what is said in the para before the example, except that it doesn't mention that it could represent a single character.

Reply:
The redundancy was on purpose, but I removed it and rephrased